### DISCUSSION TITLE
Calculator vs. non-calculator usage

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#9] [General]
One thing that we'll need to resolve is that some instructors want to use technology, and some (by choice or external policy) give assessments without any calculator/technology usage.

This issue also was pointed out by @SeppoNiemiColvin in the Calculus text [Issue 49](https://github.com/TeamBasedInquiryLearning/calculus/issues/49)

We discussed a little bit in our meeting Tuesday. I think there was consensus that the class activities can (and should) make use of technology.  As far as checkit goes, we discussed potentially just writing two checkit problems for a given outcome (e.g. in module PR, have a PR3 and a PR3T), one with no technology usage, and the latter that assumes it.

One question for @StevenClontz : A simple answer would just be to have a Checkit bank with both of those outcomes in it.  Instructors when creating assessments could just select the one they want, and tell students which one to use.  Another option, which I am unsure on implementation difficulty, would be to split into two Checkit banks--one with all the T outcomes, and one without.  Then there's a separate question of what is the cleanest presentation option (default to T, with a link to the not-T one maybe?)

**Comment by StevenClontz** [comment]
I'd lean towards a single bank, with the T alternates that assume tech use. Generally I suggest that instructors have students access the practice problems via an embedded outcome in their LMS, so there's no confusion.
